services/housetables/src/main/java/com/linkedin/openhouse/housetables/config/MainApplicationConfig.java

```diff
@@ -10,11 +11,12 @@
 /** Main application configuration to load cluster properties and define required beans */
 @Configuration
 public class MainApplicationConfig extends BaseApplicationConfig {
-  public static final String APP_NAME = "housetables";
```

Comment:
sure about no li-openhouse usage for this public static? This change doesn't seem to be very necessary regardless

Reply:
Sure will add it back.